onadata/apps/logger/models/xform.py

```diff
@@ -80,6 +80,7 @@ class XForm(BaseModel):
     uuid_bind_location = 4
     instances_with_geopoints = models.BooleanField(default=False)
     num_of_submissions = models.IntegerField(default=0)
+    attachment_storage_bytes = models.IntegerField(default=0)
```

Comment:
let's do `PositiveIntegerField` to match the `Attachment` model. hopefully adding the column will be fast when we upgrade PostgreSQL, but we should verify: https://dataegret.com/2018/03/waiting-for-postgresql-11-pain-free-add-column-with-non-null-defaults/